lib/store/actions.js

```diff
@@ -242,6 +244,42 @@ const actions = {
       },
     };
   },
+
+  startWatchingLocation() {
+    return (dispatch) => {
+      dispatch(actions.locationWatchingStarted());
+      watchID = navigator.geolocation.watchPosition(
```

Comment:
Potentially we could have two `ItemScene` mounted at once. If you open an Item then follow a notification deep link. The `Navigator` will probably keep `ItemScene`s mounted if they are in the history stack, but I don't really know. This is an implementation detail of the `Navigator`, we should be prepared for anything.